docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -38,6 +38,8 @@ \section{Problem Statement}
 
 \subsection{Problem}
 
+With the rising cost of living in Hamilton as well as across Canada, many households are searching for ways to cut back on costs of thier living expenses. Grocery prices, most notably, are quickly on the rise and are making it difficult for households to handle/manage their budgets \textbf{Grocery Spend Tracker} will strive to make it easier for users to make \textit{smart} grocery decisions to lower their spending on food and household needs.
```

Comment:
- With the rising cost of living in Hamilton as well as across Canada, many households are searching for ways to cut back on costs of **their** living expenses.
- I also think the second sentence could be broken up into two sentences:
  - Grocery prices, most notably, are quickly on the rise and are making it difficult for households to handle/manage their budgets.
  - The Grocery Spending Tracker will strive to make it easier for users to make smart grocery decisions to lower their spending on food and household needs.